vtex-apps/quickorder
# [Bugfix] Fixing the issue that when adding items in the cart, it doesn't go back to the list page in text area and upload blocks

#### What does this PR do? \*

Makes the TextArea and Upload block go back to the list page after adding products to the cart.

#### How to test it? \*

In the Text Area and Upload Block, add item to the cart and check if the component went back to the previous state

#### Describe alternatives you've considered, if any. \*

None

#### Related to / Depends on \*

None
